src/components/Slices/ConfirmAddress.jsx

```diff
@@ -110,17 +174,40 @@ const ConfirmAddress = ({ slice, network }) => {
 
   return (
     <Grid item md={12}>
-      <ExtendedPublicKeySelector number={0} onChange={keySelected} />
+      <ExtendedPublicKeySelector number={0} onChange={handleKeySelected} />
+      {state.deviceType === "unknown" && state.keySelected && (
+        <form>
+          <FormControl fullWidth>
+            <InputLabel>Select Method</InputLabel>
+
+            <Select
+              id="confirm-importer-select"
+              value=""
+              onChange={handleMethodChange}
+            >
+              <MenuItem value="">{"< Select method >"}</MenuItem>
+              <MenuItem value={TREZOR}>Trezor</MenuItem>
+              <MenuItem value={LEDGER} disabled>
```

Comment:
Obviously this isn't a blocker, but it'd be great to have some constant or a method on the keystores some way to determine programmatically support for certain interactions. then this could be something like `disabled=!keystore.hasConfirmAddress()`. Anyway, just something to think about I suppose.